.travis.yml

```
@@ -66,8 +65,7 @@ env:
 
 script:
```

```suggestion
script:
  - export CXXFLAGS="-Wall -Wextra -pedantic -Werror"
```
Add to avoid my mistakes sticking around (thanks for fixing the one in read_data)